FFCDemoPaymentService.Tests/MessagingTests/MessageServiceTests.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.ComponentModel;
+using System.IO;
+using NUnit.Framework;
+using Moq;
+using FFCDemoPaymentService;
+using FFCDemoPaymentService.Messaging;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace FFCDemoPaymentService.UnitTests.MessagingTests
+{
+
+    public class MessageServiceTests
+    {        
+        MessageConfig messageConfig;
+        MessageService messageService;
+
+        [SetUp]
+        public void Setup()
+        {
+            messageConfig = new MessageConfig();
+            messageConfig.Host = "123";
+            messageConfig.Port = 111;
+            messageConfig.UseSsl = true;
+            messageConfig.PaymentQueue = "Test_Queue";
+            messageConfig.PaymentUserName = "Test_User";
+            messageConfig.PaymentPassword = "Test_Password";
+            var iConnection = new Mock<IConnection>();
+            var serviceScopeFactory = new Mock<IServiceScopeFactory>();
+            messageService = new MessageService ( iConnection.Object, messageConfig, serviceScopeFactory.Object );
+        }
+
+        [Test]
+        public void ConfigHost()
+        {
+            Assert.AreEqual(messageConfig.Host, "123");
+        }
+
+        [Test]
+        public void ConfigPort()
+        {
+            Assert.AreEqual(messageConfig.Port, 111);
+        }
+
+        [Test]
+        public void ConfigSSL()
+        {
+            Assert.AreEqual(messageConfig.UseSsl, true);
+        }
+
+        [Test]
+        public void ConfigQueue()
+        {
+            Assert.AreEqual(messageConfig.PaymentQueue, "Test_Queue");
+        }
+
+        [Test]
+        public void COnfigUsername()
+        {
+            Assert.AreEqual(messageConfig.PaymentUserName, "Test_User");
+        }
+
+        [Test]
+        public void ConfigPassword()
+        {
+            Assert.AreEqual(messageConfig.PaymentPassword, "Test_Password");
+        }
+
+        [Test]
+        public void BrokerUrlString_NoSsl()
```

Comment:
I'd split the tests into relevant classes so they're easier to run in isolation and find the test you're after when adding or diagnosing faults.  eg. instead of `MessagingTests` have `MessageServiceTests`, `BrokerUrlTests` etc

Reply:
Split out as suggested.